auction-app-backend/src/main/java/com/example/auctionapp/repository/BidRepository.java

```diff
@@ -5,7 +5,13 @@
 import org.springframework.data.jpa.repository.Query;
 
 import java.util.UUID;
+import java.util.List;
 
 public interface BidRepository extends JpaRepository<BidEntity, UUID> {
     BidEntity findTopBidEntityByProductEntity_ProductIdOrderByBidTimeDesc(final UUID productId);
+    List<BidEntity> findBidEntitiesByUserEntity_UserId(final UUID userId);
+    @Query("SELECT b FROM BidEntity b " +
```

Comment:
Can we achieve this just from method name, can JPA help use without writing custom query?

Reply:
i tried to achieve this from the method name only but it was quite complex due to the subquery so i couldn't get the desired result